packages/flutter_tools/lib/src/commands/daemon.dart

```diff
@@ -443,6 +443,9 @@ class DaemonDomain extends Domain {
       if (featureFlags.areCustomDevicesEnabled) {
         result.add('custom');
       }
+      if (featureFlags.isPreviewDeviceEnabled && supportedPlatforms.contains(SupportedPlatform.windows)) {
+        result.add('preview');
```

Comment:
After this change, sending the following message to `flutter daemon`:

```json
[{"id":"1","method":"daemon.getSupportedPlatforms","params":{"projectRoot":"D:\\git\\hex_viewer"}}]
```

I received the following response:

```json
[{"id":"1","result":{"platforms":["windows","ios","android","web","preview"]}}]
```

Reply:
Is there a reason for this to be its own platformType, and not just use the existing one like "windows"?

Reply:
The code below actually uses `PlatformType.windows`, so I don't think this code here is doing anything. It's say "The 'preview' platform type is supported" but there are no devices with a platform type of preview, because the preview device has a platform type of "windows".

Reply:
I guess a good question is whether we should use a new platform type or existing one. If we use "windows", then we have no way to tell the IDE that this device is not supported (eg. the project uses plugins). However if we use a custom one ("preview" or "windows-preview") then we can conditionally tell the IDE individually which of them is supported - but I don't know if that has implications for the rest of Flutter (eg. if there is build logic that depends on "windows"? probably more likely it would use the real `platform`, and not `platformType`?)

Reply:
We resolved this offline, and concluded that we would add a new PlatformType.windowsPreview as this would give us the most flexibility in the future.